cpp/ql/test/library-tests/ir/ir/raw_ir.expected

```diff
@@ -107,6 +107,8 @@ bad_asts.cpp:
 #   30|     v30_6(void)         = ExitFunction            : 
 
 clang.cpp:
+#    3| int globalInt
```

Comment:
I assume that this is a global variable with no initializer? Do we also dump functions that don't have IR as well, like virtual functions with no body? It just seems odd, although not necessarily wrong.

Reply:
Yes, it is. I think you're right that these can be excluded.